android/app/build.gradle

```diff
@@ -62,18 +88,18 @@ dependencies {
 	implementation 'androidx.appcompat:appcompat:1.4.1'
 	implementation 'com.google.android.material:material:1.5.0'
 	implementation "androidx.compose.ui:ui:$compose_version"
-	implementation 'androidx.compose.material:material:1.1.0'
+	implementation 'androidx.compose.material:material:1.1.1'
 	implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
 	implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
 	implementation 'androidx.activity:activity-compose:1.4.0'
-	implementation 'androidx.compose.runtime:runtime-livedata:1.1.0'
+	implementation 'androidx.compose.runtime:runtime-livedata:1.1.1'
     implementation 'androidx.security:security-crypto:1.1.0-alpha03'
     implementation 'androidx.biometric:biometric:1.1.0'
 	implementation 'androidx.security:security-crypto-ktx:1.1.0-alpha03'
     implementation 'androidx.camera:camera-core:1.0.2'
     implementation 'androidx.camera:camera-lifecycle:1.0.2'
-	implementation 'androidx.camera:camera-view:1.1.0-beta01'
-	implementation 'androidx.camera:camera-camera2:1.1.0-beta01'
+	implementation 'androidx.camera:camera-view:1.1.0-beta02'
+	implementation 'androidx.camera:camera-camera2:1.1.0-beta02'
```

Comment:
TODO: test on several devices, check changelog. This is known to break a bit occasionally.

Reply:
actually reverted the bump, let's make this changeset minimal